esda/moran.py

```diff
@@ -1700,12 +1743,76 @@ def by_col(
             outvals=outvals,
             swapname=swapname,
             stat=Moran_Local,  # how would this get done w/super?
-            **stat_kws
+            **stat_kws,
         )
         for col in rate_df.columns:
             df[col] = rate_df[col]
 
 
+def _explore_local_moran(moran_local, gdf, crit_value, explore_kwargs=None, m=None):
```

Comment:
```suggestion
def _explore_local_moran(moran_local, gdf, crit_value, m=None, **kwargs):
```
Can we just pass everything else automatically to geopandas explore without a need to wrap in as `explore_kwargs`?

Reply:
In that case you can also remove `m` from the signature as it would be inherited.